### PR TITLE
[New] Add keycloak prod config to UAT web app

### PR DESCRIPTION
**Pull request by Mousto097**
- Update "auth-server-url" in both Fontend and Backend keycloak.json files for UAT web app to use Keycloak PROD Server
- Fixed issue when a user have no roles in keycloak admin console

### CODE REVIEW

File: services/backend/util/keycloak.js [hunk 1]
```diff
@@ -8,10 +8,10 @@ const keycloak = new Keycloak({ store: memoryStore });
 require("dotenv").config();
 
 const sessionInstance = session({
-  secret: process.env.KEYCLOAK_SECRET,
+  secret: process.env.DEV_KEYCLOAK_SECRET,
```
Comment (Rizvida1) [inline services/backend/util/keycloak.js]: This is not how it should be done

**Review by Rizvida1 — CHANGES_REQUESTED** [review]

File: services/backend/util/keycloak.js (hunk 1 above)
Comment (Mousto097) [reply services/backend/util/keycloak.js]: I'm not done yet! My PR is still not ready for review

File: env/web.env.example [hunk 2]
```diff
@@ -1,2 +1,3 @@
 REACT_APP_API_ADDRESS=http://localhost:8080/
-REACT_APP_API_ADDRESS_DEV=http://mytalent-backend-dev-mytalent.apps.dev.openshift.ised-isde.canada.ca/
\ No newline at end of file
+REACT_APP_API_ADDRESS_DEV=http://mytalent-backend-dev-mytalent.apps.dev.openshift.ised-isde.canada.ca/
+REACT_APP_KEYCLAOK_AUTH_SERVER_URL=[KEYCLAOK_AUTH_SERVER_URL=[KEYCLAOK_AUTH_SERVER_URL]
```
Comment (Rizvida1) [inline env/web.env.example]: Keycloak is misspelled

**Review by Rizvida1 — CHANGES_REQUESTED** [review]

File: env/web.env.example (hunk 2 above)
Comment (Mousto097) [reply env/web.env.example]: I fixed all mispelling errors.

File: services/frontend-v3/src/routes/Secured.js [hunk 3]
```diff
@@ -49,16 +55,16 @@ function Secured(props) {
...
         onLoad: "login-required",
         promiseType: "native",
-        checkLoginIframe: false
+        checkLoginIframe: false,
       })
-      .then(authenticated => {
+      .then((authenticated) => {
         // check if user is admin
-        if (keycloak.tokenParsed.resource_access) {
+        if (keycloak.tokenParsed.resource_access["upskill-client"]) {
```
Comment (Rizvida1) [inline services/frontend-v3/src/routes/Secured.js:67]: Does this not break the code for users without roles?

**Review by Rizvida1 — CHANGES_REQUESTED** [review]

File: services/frontend-v3/src/routes/Secured.js (hunk 3 above)
Comment (Mousto097) [reply services/frontend-v3/src/routes/Secured.js:67]: No, this is the fix for the error. I already tested it!

 `if (keycloak.tokenParsed.resource_access) {...}` --> this one break the code

`if (keycloak.tokenParsed.resource_access["upskill-client"]) {...}` --> this is the fix for it.

**Review by Rizvida1 — APPROVED** [review]

**Pull request merged**